app/elements/demo-tabs.html

```diff
@@ -97,22 +120,88 @@
        */
       name: {
         type: String
-      }
+      },
+
+      editorOpenFile: {
+        type: String,
+        value: 'index.html',
+      },
+
+      /**
+       * absolute path to the project directory
+       *
+       * e.g. /3.0/start/samples/custom-element
+       *     or /3.0/start/samples/employee-list
+       */
+        projectPath: {
+        type: String,
+      },
+
+      _displaySelected: {
+        type: String,
+        value: 'static',
+      },
+
+      _isPlunkr: {
+        type: Boolean,
+      },
+
+      _isStackBlitz: {
+        type: Boolean,
+      },
```

Comment:
`_isPlunkr` and `_isStackBlitz` unused

Reply:
ah I forgot to rename `srcDefined` to this etc.